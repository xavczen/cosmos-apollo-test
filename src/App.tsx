import React, { Component } from 'react';
import RequestData from './RequestData';

interface IProps {
  renderMode: string;
  baseURL: string;
}

class App extends Component<IProps, any> {
  public render() {
    return (
      <RequestData baseURL={this.props.baseURL} />
    );
  }
}

export default App;
