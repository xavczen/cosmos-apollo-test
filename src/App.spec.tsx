import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from './App';

describe('when render mode is request_data', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(
      <App renderMode="request_data" baseURL="whatever" />
    );
    expect(tree).toMatchSnapshot();
  });
});
