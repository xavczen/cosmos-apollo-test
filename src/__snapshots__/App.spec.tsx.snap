// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when render mode is request_data renders correctly 1`] = `
<Unknown
  baseURL="whatever"
/>
`;
